org.mixedrealitytoolkit.spatialmanipulation/Solvers/Solver.cs

```diff
@@ -218,6 +229,11 @@ protected virtual void Awake()
             }
 
             GoalScale = maintainScaleOnInitialization ? transform.localScale : Vector3.one;
+
+            if (trackedPoseDriverLookup == null)
```

Comment:
Why we doing this in `Awake` and `Start`?   I suggest follow the existing pattern used by `controllerLookup`, and only do this in `Start`

Reply:
Right, that's a remnant of a first attempt that I forgot to remove, thx for catching that.  Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/12cc9a84d29f25f7250686ede287c6d10e7726b9 , resolving.